src/mono/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -643,5 +643,10 @@
                     <!-- hot_reload.c -->
                     <method name="GetInstanceFieldFieldStore" />
                 </type>
+
+                <!-- FIXME: only if feature="System.StartupHookProvider.IsSupported" featurevalue="true" -->
```

Comment:
Something like:
```suggestion
  </assembly>

	<assembly fullname="System.Private.CoreLib" feature="System.StartupHookProvider.IsSupported" featurevalue="true" featuredefault="true">
```
?

Reply:
featuredefault is what I was missing. Thanks!